.glitch-assets

```diff
@@ -125,3 +125,9 @@
 {"name":"icon_external.png","date":"2020-03-12T17:33:16.451Z","url":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2Ficon_external.png","type":"image/png","size":284,"imageWidth":48,"imageHeight":48,"thumbnail":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2Ficon_external.png","thumbnailWidth":48,"thumbnailHeight":48,"uuid":"uu8bnjx4SkXSFLc4"}
+{"name":"sleeping-symbol_1f4a4.png","date":"2020-06-09T20:30:38.409Z","url":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2Fsleeping-symbol_1f4a4.png","type":"image/png","size":567,"imageWidth":64,"imageHeight":64,"thumbnail":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2Fsleeping-symbol_1f4a4.png","thumbnailWidth":64,"thumbnailHeight":64,"uuid":"SGbV1Luj3soie8rl"}
+{"name":"high-voltage-sign_26a1.png","date":"2020-06-09T20:30:41.714Z","url":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2Fhigh-voltage-sign_26a1.png","type":"image/png","size":833,"imageWidth":64,"imageHeight":64,"thumbnail":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2Fhigh-voltage-sign_26a1.png","thumbnailWidth":64,"thumbnailHeight":64,"uuid":"RNU1ztiVLcDAOVBj"}
+{"name":"rocket.png","date":"2020-06-12T18:14:40.393Z","url":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2Frocket.png","type":"image/png","size":5958,"imageWidth":64,"imageHeight":64,"thumbnail":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2Frocket.png","thumbnailWidth":64,"thumbnailHeight":64,"uuid":"7689z6Q0uv7OoN2z"}
+{"name":"app-dashboard-promo-background.svg","date":"2020-07-10T05:55:37.842Z","url":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2Fapp-dashboard-promo-background.svg","type":"image/svg+xml","size":5455,"imageWidth":455,"imageHeight":172,"thumbnail":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2Fthumbnails%2Fapp-dashboard-promo-background.svg","thumbnailWidth":330,"thumbnailHeight":125,"uuid":"R4OD65aQW2S8Vip4"}
+{"name":"Glitch-Error-Rainbow-Mug-hires.png","date":"2020-07-23T05:20:53.593Z","url":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2FGlitch-Error-Rainbow-Mug-hires.png","type":"image/png","size":170377,"imageWidth":2064,"imageHeight":1247,"thumbnail":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2Fthumbnails%2FGlitch-Error-Rainbow-Mug-hires.png","thumbnailWidth":330,"thumbnailHeight":200,"uuid":"JwJpFAHr0wTqi4Zf"}
+{"name":"gift.png","date":"2020-08-04T14:19:53.942Z","url":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2Fgift.png","type":"image/png","size":6712,"imageWidth":64,"imageHeight":64,"thumbnail":"https://cdn.glitch.com/d7f4f279-e13b-4330-8422-00b2d9211424%2Fgift.png","thumbnailWidth":64,"thumbnailHeight":64,"uuid":"fOT0pySgjZNRue3E"}
```

Comment:
Hrm, when I do PRs off a local copy instead of a remix I don't include asset changes, do you think that's a problem? Having them in the git repo doesn't really affect anything one way or another.

Reply:
yeah I'm not sure tbh, I just pulled this in because I had put the asset in there, but I agree it doesn't seem super critical. I cannot wait until assets are just files in projects ahhh